Commenter.Extension/Commenter.Extension.csproj

```diff
@@ -162,6 +162,10 @@
       <Private>False</Private>
       <HintPath>..\packages\VSSDK.OLE.Interop.7.0.4\lib\net20\Microsoft.VisualStudio.OLE.Interop.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Settings, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=x86">
```

Comment:
:memo: Both references to this library work in combination to prevent the library from getting included in the VSIX file.